pyart/graph/radarmapdisplay_cartopy.py

```diff
@@ -0,0 +1,510 @@
+            min_lon=None, max_lon=None, min_lat=None, max_lat=None,
+            width=None, height=None, lon_0=None, lat_0=None,
+            resolution='110m', shapefile=None, edges=True, gatefilter=None,
+            basemap=None, filter_transitions=True, embelish=True,
+            ticks=None, ticklabs=None, **kwargs):
+        """
+        Plot a PPI volume sweep onto a geographic map.
+
+        Additional arguments are passed to Basemap.
+
+        Parameters
+        ----------
+        field : str
+            Field to plot.
+        sweep : int, optional
+            Sweep number to plot.
+
+        Other Parameters
+        ----------------
+        mask_tuple : (str, float)
+            Tuple containing the field name and value below which to mask
+            field prior to plotting, for example to mask all data where
+            NCP < 0.5 set mask_tuple to ['NCP', 0.5]. None performs no masking.
+        vmin : float
+            Luminance minimum value, None for default value.
+            Parameter is ignored is norm is not None.
+        vmax : float
+            Luminance maximum value, None for default value.
+            Parameter is ignored is norm is not None.
+        norm : Normalize or None, optional
+            matplotlib Normalize instance used to scale luminance data.  If not
+            None the vmax and vmin parameters are ignored.  If None, vmin and
+            vmax are used for luminance scaling.
+        cmap : str or None
+            Matplotlib colormap name. None will use the default colormap for
+            the field being plotted as specified by the Py-ART configuration.
+        mask_outside : bool
+            True to mask data outside of vmin, vmax.  False performs no
+            masking.
+        title : str
+            Title to label plot with, None to use default title generated from
+            the field and tilt parameters. Parameter is ignored if title_flag
+            is False.
+        title_flag : bool
+            True to add a title to the plot, False does not add a title.
+        colorbar_flag : bool
+            True to add a colorbar with label to the axis.  False leaves off
+            the colorbar.
+        ticks : array
+            Colorbar custom tick label locations.
+        ticklabs : array
+            Colorbar custom tick labels.
+        colorbar_label : str
+            Colorbar label, None will use a default label generated from the
+            field information.
+        ax : Axis
+            Axis to plot on. If provided, must have a Cartopy crs projection.
+        fig : Figure
+            Figure to add the colorbar to. None will use the current figure.
+        lat_lines, lon_lines : array or None
+            Locations at which to draw latitude and longitude lines.
+            None will use default values which are resonable for maps of
+            North America.
+        projection : cartopy.crs class
+            Map projection supported by cartopy.
```

Comment:
Add text indicating that the default is a Lambert conformal projection centered at lat_0.